app/frontend/actions.ts

```diff
@@ -1139,7 +1140,7 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     resetAccountIndexes(state)
     setState({
       sendAddress: {fieldValue: ''},
-      sendAmount: {fieldValue: '', coins: 0},
+      sendAmount: {fieldValue: '', coins: 0 as Lovelace},
```

Comment:
This exact code appears 7 times in actions, this would also be a good candidate for a separate function, e.g., 
```
const initialAmountField () => ({fieldValue: '', coins: 0 as Lovelace})
```

Reply:
I agree, but maybe not setting it 7 places is the solution and we should reset it only in one place and call that function everywhere where needed.. Also as mentioned I typed only the necessary to move on with tx planning refactor not addressing other refactors.